lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -499,8 +499,11 @@ bool SessionStateEnforcer::static_rule_init() {
   fd1.mutable_match()->set_direction(FlowMatch_Direction_UPLINK);
   fd1.set_action(FlowDescription_Action_PERMIT);
   rule1.mutable_flow_list()->Add()->CopyFrom(fd1);
-  reqpdr1.mutable_activate_flow_req()->mutable_dynamic_rules()->Add()->CopyFrom(
-      rule1);
+  VersionedPolicy versioned_rule1;
+  versioned_rule1.set_version(reqpdr1.pdr_version());
+  versioned_rule1.mutable_rule()->CopyFrom(rule1);
+  reqpdr1.mutable_activate_flow_req()->mutable_policies()->Add()->CopyFrom(
+      versioned_rule1);

```

Comment:
so that means that even 5g pdr are using the same container for policies than 4g?

Reply:
Looks like it. I messaged @gurrapualt to confirm.